src/Stratis.Bitcoin.Features.Wallet/WalletRPCController.cs

```diff
@@ -261,6 +261,73 @@ public UnspentCoinModel[] ListUnspent(int minConfirmations = 1, int maxConfirmat
+        public async Task<uint256> SendManyAsync(string fromAccount, string addressesJson, int minConf = 1, string comment = null, string subtractFeeFromJson = null, bool isReplaceable = false, int? confTarget = null, string estimateMode = "UNSET")
+        {
+            var accountReference = this.GetAccount();
+           
+            // Optional list of addresses to subtract fees from.
+            IEnumerable<BitcoinAddress> subtractFeeFromAddresses = null;            
+            if (!string.IsNullOrEmpty(subtractFeeFromJson))
+            {
+                subtractFeeFromAddresses = JsonConvert.DeserializeObject<List<string>>(subtractFeeFromJson).Select(i => BitcoinAddress.Create(i, this.fullNode.Network));
+            }
+
+            // Outputs addresses are keyvalue pairs of address, amount. Translate to Receipient list.
+            IEnumerable<Recipient> recipients = null;
+            if (!string.IsNullOrEmpty(addressesJson))
+            {
+                recipients = JsonConvert.DeserializeObject<Dictionary<string, decimal>>(addressesJson).Select(address => new Recipient
+                {
+                    ScriptPubKey = BitcoinAddress.Create(address.Key, this.fullNode.Network).ScriptPubKey,
+                    Amount = Money.Coins(address.Value),
+                    SubtractFeeFromAmount = subtractFeeFromAddresses == null ? false : subtractFeeFromAddresses.Contains(BitcoinAddress.Create(address.Key, this.fullNode.Network))
+                });
+            }
+
+            if (recipients == null || !recipients.Any())
+            {
+                throw new RPCServerException(RPCErrorCode.RPC_INVALID_PARAMETER, "No valid output addresses specified.");
+            }
+          
+            var context = new TransactionBuildContext(this.fullNode.Network)
+            {
+                AccountReference = accountReference,                
+                MinConfirmations = minConf,
+                Shuffle = true, // We shuffle transaction outputs by default as it's better for anonymity.                
+                Recipients = recipients.ToList()                
+            };
+
+            // Set fee type for transaction build context.
+            context.FeeType = FeeType.Medium;
+            if (estimateMode.Equals("ECONOMICAL", StringComparison.InvariantCultureIgnoreCase))
+                context.FeeType = FeeType.Low;
+            else if (estimateMode.Equals("CONSERVATIVE", StringComparison.InvariantCultureIgnoreCase))
+                context.FeeType = FeeType.High;
+
+            try
+            { 
+                // Log warnings for currently unsupported parameters.
+                if (!string.IsNullOrEmpty(comment))
+                    this.logger.LogWarning("'comment' parameter is currently unsupported. Ignored.");
+                if (isReplaceable)
+                    this.logger.LogWarning("'replaceable' parameter is currently unsupported. Ignored.");
```

Comment:
Use `nameof` here as well.

Reply:
replaceable is the name of the rpc parameter in rpc docs. The parameter name in this method doesn't matter as the order the parameters are passed is how it determines which one is which.